scene/resources/world_2d.cpp

```diff
@@ -387,7 +386,7 @@ World2D::World2D() {
 
 	//set space2D to be more friendly with pixels than meters, by adjusting some constants
 	Physics2DServer::get_singleton()->space_set_active(space, true);
-	Physics2DServer::get_singleton()->area_set_param(space, Physics2DServer::AREA_PARAM_GRAVITY, GLOBAL_DEF("physics/2d/default_gravity", 98));
+	Physics2DServer::get_singleton()->area_set_param(space, Physics2DServer::AREA_PARAM_GRAVITY, GLOBAL_DEF("physics/2d/default_gravity", 9.8));
```

Comment:
No it shouldn't be changed; and actually I think it would be best to revert to the previous behaviour, unless we decide to refactor the 2D physics to work with "natural" values and not ones multiplied by 10.

Here we're now hiding from the user the fact that the engine actually expects 98 for a "life-like" gravity, so users overriding gravity from script would end up surprised when setting e.g. "3" instead of "30".

So if you can revert to your original commit, I think that's best. Sorry for the unnecessary additional work :)